dsl/src/test/scala/org/http4s/dsl/PathInHttpRoutesSuite.scala

```diff
@@ -294,4 +300,25 @@ class PathInHttpRoutesSuite extends Http4sSuite {
     response.map(_.status).assertEquals(Ok) *>
       response.flatMap(_.as[String]).assertEquals("flag not present")
   }
+  test("Path DSL with -->> should match on GET") {
```

Comment:
The new operator is `->>`, but the tests refer to `-->>`.

Reply:
Oops, that's what I get for using coding ligatures.